api/src/application/user/user-service.js

```diff
@@ -101,7 +101,7 @@ module.exports = (bookshelf, emailService, renderTemplate) => {
     },
 
     findByEmailForOidc: function(email) {
-      return bookshelf.model('user').forge({ email }).fetch()
+      return bookshelf.model('user').where('email', email).fetch()
```

Comment:
`.forge()` was throwing an error like this:
`api_1       | 170620/182352.571, [error] message: Undefined binding(s) detected when compiling SELECT query: select "SIP_user".* from "SIP_user" where "SIP_user"."email" = ? and "SIP_user"."profile" = ? and "SIP_user"."app_metadata" = ? limit ?, stack: Error: Undefined binding(s) detected when compiling SELECT query: select "SIP_user".* from "SIP_user" where "SIP_user"."email" = ? and "SIP_user"."profile" = ? and "SIP_user"."app_metadata" = ? limit ?`

After this change, I had no issues.